pkgs/by-name/tp/tproxy/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "tproxy";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "kevwan";
+    repo = "tproxy";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-LX3ciC3cMYuQPm6ekEkJPrSdTXH+WZ4flXyrsvJZgn8=";
+  };
+
+  vendorHash = "sha256-7s2gnd9UR/R7H5pcA8NcoenaabSVpMh3qzzkOr5RWnU=";
+
+  ldflags = [
+    "-w"
+    "-s"
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "CLI tool to proxy and analyze TCP connections";
+    homepage = "https://github.com/kevwan/tproxy";
+    changelog = "https://github.com/kevwan/tproxy/releases/tag/v${version}";
```

Comment:
As far as remember, @Pandapip1 commented that parameterized links are discouraged because they are fragile - if a future version string changes to the `0-unstable-YYYYMMDD` format, then this link would break.

Reply:
Uh, that's true. In that case, [the wiki](https://nixos.org/manual/nixpkgs/stable/#var-meta-changelog) has a wrong example :sweat_smile:

Reply:
I see. Should I remove the changelog attribute then?

Reply:
It's the same as 4915 other packages. If someone actually wants to change it, it really should be documented properly / publicly - this feels like a wrong place as a one-off.
It can be mass-changed once there's consensus.

Reply:
> It's the same as 4915 other packages. If someone actually wants to change it, it really should be documented properly / publicly - this feels like a wrong place as a one-off.

See discussion at https://github.com/NixOS/nixpkgs/issues/277994

Reply:
That's kinda fine. We want a direct link to the change log if possible and we just need to be careful when updating to unstable versions.